<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_AsmConfig</class>
 <widget class="QDialog" name="Dial_AsmConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asm Config</string>
  </property>
  <widget class="QCheckBox" name="checkBox_All">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_0">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 9</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 11</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Clear">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Asm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Fname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Do">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Do It</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Select File</string>
   </property>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>26</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>451</width>
     <height>79</height>
    </rect>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>checkBox_All</zorder>
  <zorder>checkBox_Asm_0</zorder>
  <zorder>checkBox_Asm_1</zorder>
  <zorder>checkBox_Asm_2</zorder>
  <zorder>checkBox_Asm_3</zorder>
  <zorder>checkBox_Asm_4</zorder>
  <zorder>checkBox_Asm_5</zorder>
  <zorder>checkBox_Asm_6</zorder>
  <zorder>checkBox_Asm_7</zorder>
  <zorder>checkBox_Asm_8</zorder>
  <zorder>checkBox_Asm_9</zorder>
  <zorder>checkBox_Asm_11</zorder>
  <zorder>checkBox_Asm_10</zorder>
  <zorder>checkBox_Clear</zorder>
  <zorder>lineEdit_Fname</zorder>
  <zorder>pushButton_Do</zorder>
  <zorder>pushButton_Close</zorder>
  <zorder>progressBar</zorder>
  <zorder>textEdit_Msg</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
